### PR TITLE
Feature/chat brand pivot

### PR DESCRIPTION
**Pull request by MathiasOhrstrom**

### CODE REVIEW

File: app/src/main/java/com/hedvig/app/util/extensions/ContextExtensions.kt [hunk 1]
```diff
@@ -141,6 +142,18 @@ fun Context.showAlert(
         }
         .show()
         .apply {
+            getButton(AlertDialog.BUTTON_POSITIVE).apply {
```
Comment (hedvigoscar) [inline app/src/main/java/com/hedvig/app/util/extensions/ContextExtensions.kt]: This is probably not a good way to theme an AlertDialog 😞

I think that by default AlertDialogs use the theme provided in the attribute `?alertDialogTheme`, check that out!

File: app/src/main/res/layout/activity_chat.xml [hunk 2]
```diff
@@ -2,6 +2,8 @@
 <androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
+    android:theme="@style/Hedvig.Theme.Chat"
```
Comment (hedvigoscar) [inline app/src/main/res/layout/activity_chat.xml]:
```suggestion
```

This should be specified in the `AndroidManifest.xml`.

If not automatically applied in the editor, check if the `tools:context` attribute is set 🙂

File: app/src/main/res/values/theme.xml [hunk 3]
```diff
@@ -63,4 +63,20 @@
     <attr name="colorButton" format="reference" />
     <attr name="textColorButton" format="reference" />
 
+    <style name="Hedvig.Theme.Chat" parent="Hedvig.Theme">
+        <item name="android:colorBackground">@color/colorBackgroundChat</item>
```
Comment (hedvigoscar) [inline app/src/main/res/values/theme.xml]: This seems suspect. Is the background color of the chat inconsistent with the rest of the application?

Comment (MathiasOhrstrom) [reply app/src/main/res/values/theme.xml]: Yes, the dark background on the chat screen is #1B1B1B and the others are #121212. Maybe I should talk to Zak about that

Comment (hedvigoscar) [reply app/src/main/res/values/theme.xml]: Yes, go 👮

File: app/src/main/java/com/hedvig/app/util/extensions/ContextExtensions.kt [hunk 4]
```diff
@@ -127,34 +125,20 @@ fun Context.showAlert(
...
-            getButton(AlertDialog.BUTTON_NEGATIVE).apply {
-                background = ColorDrawable(Color.TRANSPARENT)
-                setTextColor(resources.getColor(R.color.purple))
-            }
-            getButton(AlertDialog.BUTTON_NEUTRAL).apply {
-                background = ColorDrawable(Color.TRANSPARENT)
-                setTextColor(resources.getColor(R.color.purple))
-            }
-            window?.setBackgroundDrawable(ColorDrawable(Color.TRANSPARENT))
             getButton(AlertDialog.BUTTON_POSITIVE)?.isAllCaps = false
```
Comment (hedvigoscar) [inline app/src/main/java/com/hedvig/app/util/extensions/ContextExtensions.kt]: While we're at it, let's go all the way with theming and use the theme to control this as well.

**Pull request merged**